Apps/WebClient/src/Server/Models/DependentModel.cs

```diff
@@ -0,0 +1,41 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// Represents a Dependent Model.
+    /// </summary>
+    public class DependentModel
+    {
+        /// <summary>
+        /// Gets or sets the name of the dependent.
+        /// </summary>
+        [JsonPropertyName("name")]
+        public string Name { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the hdId of the dependent.
+        /// </summary>
+        [JsonPropertyName("hdId")]
+        public string HdId { get; set; } = string.Empty;
```

Comment:
We do need to return the gender and date of birth too!

Reply:
and a masked PHN!

Reply:
adding gender and DoB.
@tiagograf-AOT why do we need the masked PHN?

Reply:
should that not be on the next story? this is about adding a dependant right?

Reply:
my bad, I thought this was about adding the dependant. In the future lets please finish one story/task first before moving onto the next one

Reply:
I think we need to discuss this at our retros as I don't think it is productive if we have to wait and do nothing while waiting for a complete review. Mbe it is better to somehow to get the context of the story down to the PR level to make it easier for reviewers.

Reply:
I will add the PHN to the DependentModel as a part of this task related to Controller.
The masking PHN should be a new task and be done at the Service level right?

Reply:
right leo!

Reply:
perfect, thanks @tiagograf-AOT